voice/bxml/verbs/ring.md

```diff
@@ -8,6 +8,7 @@ The Ring verb is used to play ringing audio on a call.
 | ATTRIBUTE | Description                                                                                            |
 |:----------|:-------------------------------------------------------------------------------------------------------|
 | duration  | (optional) How many seconds to play ringing on the call. Default value is 5. Range: decimal values between 0.1 - 86400.
+| answerCall  | (optional) A boolean indicating whether or not to pick up the call when Ring is the first verb on an unanswered incoming call. Default value is true.
```

Comment:
I feel like this needs a better description. I know this is basically getting into the differences between common usage for "pick up a call" and SIP usage, but to me these reads as sort of nonsense. Why would we default to picking up an unanswered call? What does this actually change about the behavior of having a Ring verb? I don't have a good suggestion for the change because I never quite understood what the ask for this story was, but at least to me not having any context, I think we want a better description here.

Reply:
Yes. It'll change from 180 with `answerCall` to false and 200 with true.

Reply:
I would just change `pick up` to `answer` and make the change I suggested in my other comment